.github/workflows/test.yml

```
@@ -96,4 +96,18 @@ jobs:
         with:
           name: output-log-file
           path: testcontainers-client.log
-          retention-days: 30
\ No newline at end of file
+          retention-days: 30
+
+  with-project-keys:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Testcontainers Cloud Client with project-key and project-url
+        uses: ./
+        with:
+          version: next
+          token: ${{ secrets.TC_CLOUD_TOKEN }}
+          project-key: test
```

```suggestion
          project-key: testcontainers-cloud-setup-action-custom
```

Would it make sense to be more specific? To avoid a weird-looking `test` in the dashboard in case we list it somehow